Formula/micropython.rb

```diff
@@ -32,11 +32,17 @@ def install
   end
 
   test do
+    lib_extension = "dylib"
+
+    on_linux do
+      lib_extension = "so.6"
+    end
```

Comment:
Great suggestions for this.  I wasn't sure how we handle this (hopefully rare) case where the soname needs to be versioned but this will be a good example for the future.